udata/core/dataset/models.py

```diff
@@ -697,7 +692,7 @@ def update_resource(self, resource):
         }
         self.update(**data)
         self.reload()
-        post_save.send(self.__class__, document=self)
+        self.on_resource_updated.send(document=self, resource_id=resource.id)
 
     def remove_resource(self, resource):
```

Comment:
Don't we want a `on_resource_deleted` signal as well? Or should we add it only when we need it?

Reply:
Maybe it's better to keep as few signals as possible?